### PR TITLE
feat(examples): add examples in JSON-LD and JWT format

### LINKED ISSUE #11: Add example files
The repository should contain example DID documents and presentations.

### PR DESCRIPTION
**Pull request by jceb**
Closes #11 
Changes the example to a simple business registration document, which addresses #19

### CODE REVIEW

**Comment by jceb** [comment]
This PR builds on PR #28

**Comment by jceb** [comment]
I took the code from the [did-configuration](https://github.com/decentralized-identity/.well-known/) spec repository. Not sure whether this should be referenced somewhere.

File: resources/linked-vp/generate-config/__fixtures__/linked-vp.json [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+            "schema:streetAddress": "1 Example Street"
+          }
+        },
+        "proof": {
+          "type": "Ed25519Signature2018",
+          "created": "2024-01-05T09:31:51Z",
+          "verificationMethod": "did:example:123#_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A",
+          "proofPurpose": "assertionMethod",
+          "jws": {
+            "0": 72,
```
Comment (brianorwhatever) [inline resources/linked-vp/generate-config/__fixtures__/linked-vp.json]: This should be a string 🤔

**Review by brianorwhatever — CHANGES_REQUESTED** [review]
A lot of the examples seem to have byte arrays where they should be strings

File: resources/linked-vp/generate-config/__fixtures__/linked-vp.json (hunk 1 above)
Comment (jceb) [reply resources/linked-vp/generate-config/__fixtures__/linked-vp.json]: Hm, interesting. The byte array was returned by vc.js. I'll take a look at it.

Comment (jceb) [reply resources/linked-vp/generate-config/__fixtures__/linked-vp.json]: Fix, apparently the vc.js lib doesn't properly generate a string.

**Pull request merged**